testgrid/config/config_test.go

```diff
@@ -183,6 +214,22 @@ func TestConfig(t *testing.T) {
 			t.Errorf("DashboardGroup %v: - DashboardGroup must have a name", idx)
 		}
 
+		found := false
+		for _, kind := range prefixes {
+			for _, prefix := range kind {
+				if strings.HasPrefix(dashboardGroup.Name, prefix+"-") || prefix == dashboardGroup.Name {
+					found = true
+					break
+				}
+			}
+			if found {
+				break
+			}
+		}
+		if !found {
+			t.Errorf("Dashboard group %v: must prefix with one of: %v", dashboardGroup.Name, prefixes)
+		}
+
 		// All dashboardgroup must not have duplicated names
```

Comment:
Can this enforce two additional/stronger constraints?

1) No dashboard group OR dashboard can have the same name
2) No dashboard can be in more than one group.

Reply:
1. Done
2. enforced by L243